CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # buildah-build Changelog
 
+## v2.1
+- Add input `archs` to allow you to build image(s) with multiple architecture support [803a141](https://github.com/redhat-actions/buildah-build/commit/803a1413e7c2a594cbfb6680bca358bfdbe36745)
```

Comment:
how about 

Add `archs` input to allow building images for custom architectures.

Reply:
Better :+1: